givitsite/friendreq/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
-
+from .models import ItemRequest
 # Register your models here.
+
+admin.site.register(ItemRequest)
```

Comment:
please fix the newline

Reply:
i will answer the 2 last issues- 
1. about the install postgres we testing everthing and update the bootstrap.sh with all change that needed you can see it in my pull request - https://github.com/beyond-io/givit/pull/9 we also upadate some guide needed in the readme.md. 2. about the decision to move to postgres i will add it to the readme.md..

last question- the seperation you exepect that we change this current PR or for the future? 

thanks 👍

Reply:
Please make sure you explicitly state the dependency on PR #9 in the PR description so the PRs will be merged in the right order, unfortunately GitHub cannot enforce it (There are systems that can).

I expect the separation to happen for this PR. 
Here is how to do it with Git: https://embeddedartistry.com/blog/2018/02/19/code-cleanup-splitting-up-git-commits-in-the-middle-of-a-branch/

Reply:
Thanks for all the notes.
We will do it an re - pull